src/templates/vispro_terms.tsv

```diff
@@ -0,0 +1,16 @@
+id	label	definition	definition source URI	type	super class	super property	inverse property	domain (if property)	range (if property)	term creaor URI (e.g. ORCID)	term curator URI (e.g. ORCID)	example of usage	editor note
+ID	AL rdfs:label@en	AL IAO:0000115@en	>AI dc11:source	TYPE	SC % SPLIT=|	SP %	IP %	DOMAIN	RANGE	AI dc11:creator	AI IAO:0000117	AL IAO:0000112@en	AL IAO:0000116@en
+obo:VISPRO_0000000	vibrational spectrometer configuration specification	A setting datum that specifies the needed settings of a spectrometer used in vibrational spectroscopy.		owl:class	setting datum 					ORCID:0000-0002-1595-3213			
```

Comment:
try owl:Class instead of owl:class. I think robot does not recognize owl:class, creates a custom class in the OWL namespace and makes your classes instances of it.

Reply:
well WTF that was it! Muchos gracias!!!